memgpt/cli/cli.py

```diff
@@ -437,12 +441,16 @@ def run(
             context_window=context_window,
         )
 
-        # TODO: allow configrable state manager (only local is supported right now)
-        persistence_manager = LocalStateManager(agent_config)  # TODO: insert dataset/pre-fill
-
         # save new agent config
         agent_config.save()
-        typer.secho(f"Created new agent {agent_config.name}.", fg=typer.colors.GREEN)
+        typer.secho(f"->  🤖 Using persona profile '{agent_config.persona}'", fg=typer.colors.WHITE)
+        typer.secho(f"->  🧑 Using human profile '{agent_config.human}'", fg=typer.colors.WHITE)
+        typer.secho(f"🎉 Created new agent '{agent_config.name}'", fg=typer.colors.GREEN)
+
+        # Supress llama-index noise
+        with suppress_stdout():
+            # TODO: allow configrable state manager (only local is supported right now)
+            persistence_manager = LocalStateManager(agent_config)  # TODO: insert dataset/pre-fill
```

Comment:
Moved persistence_manager assignment after - @sarahwooders I don't think this changes anything though? Can also move back (was just moving it around to ID where the stray MockLLM print statement was coming from)